src/test/java/edu/gwu/csci6212/MinimumStepsChessKnightTest.java

```diff
@@ -0,0 +1,56 @@
+package edu.gwu.csci6212;
+
+public class MinimumStepsChessKnightTest {
+
+
+    public static void  outOfBoad(){
```

Comment:
I think this is a typo and you meant outOfBoard? I personally think outOfBounds might be a bit more clear. It sounds like soccer and all the sports ⚽

Reply:
Oh and tests are usually a bit more verbose so they say what the expected behavior is in the test. E.g. throwsOutOfBoundExceptionWhenPassedCoordinatesOutOfBounds. Some folks don't like this, but I usually try to be super verbose so my tests and code are easy to understand